Gemfile

```diff
@@ -2,3 +2,5 @@ source "https://rubygems.org"
 
 # Specify your gem's dependencies in cacheable.gemspec
 gemspec
+
+gem 'rails', '~> 6.0.0'
```

Comment:
I tried to remove the dependency on rails in #27 so you can use it in Rack apps without having to include rails as a dependency.

Reply:
This is not a runtime dependency on Rails. It is a development dependency on Rails to make sure it works in Rails applications. It is only used in the test suite.

Reply:
Just to be clear I didn't change the dependencies of the gem. I just made explicit in the Gemfile what was already in the gemspec as development dependency so we could chose which version of Rails the gem was tested. We should also have a test suite without Rails so we make sure it works great for rack only apps. I'll work on that.